CHANGELOG.md

```diff
@@ -1,6 +1,19 @@
 ## This library follows [Semantic Versioning](https://semver.org).
 ## This CHANGELOG follows [keepachangelog](https://keepachangelog.com/en/1.0.0/).
 
+###  VERSION 3.0.0
```

Comment:
Would you mind adding a link to the PR?

Add this at the end of the file:
```
[3.0.0]: https://github.com/AppsFlyer/aerospike-clj/pull/62
```

And a link here:
```suggestion
###  VERSION [3.0.0]
```